R2API/Director/DirectorAPIhelpers.cs

```diff
@@ -400,6 +408,41 @@ public static class InteractableNames {
                 };
             }
 
+            /// <summary>
+            /// Adds a new monster to matching stages.
+            /// Also add to each existing monster families if second parameter is true.
+            /// For custom stages use Stage.Custom and enter the name of the stage in customStageName.
+            /// </summary>
+            /// <param name="monsterCard"></param>
+            /// <param name="addToFamilies"></param>
+            /// <param name="matchStage"></param>
+            public static void AddNewMonsterToStagesWhere(DirectorCardHolder monsterCard, bool addToFamilies, Predicate<StageInfo> matchStage) {
+                ThrowIfNotLoaded();
+
+                MonsterActions += (dccsPool, mixEnemyArtifactMonsters, currentStage) => {
+                    if (matchStage(currentStage)) {
+                        AddNewMonster(dccsPool, mixEnemyArtifactMonsters, monsterCard, addToFamilies);
+                    }
+                };
+            }
+
+            /// <summary>
+            /// Adds new monsters based on a selector.
+            /// First parameter is the current stage.
+            /// First component of a return tuple is the monster card holder.
+            /// Second component of a return tuple is whether to also add to existing monster families.
+            /// </summary>
+            /// <param name="monstersSelector"></param>
+            public static void AddSelectedNewMonsters(Func<StageInfo, IEnumerable<Tuple<DirectorCardHolder, bool>>> monstersSelector) {
```

Comment:
Seem really awkward to call? If one already has an ienumerable of directorCardHolders, you could iterate over  AddNewMonsterToStagesWhere yorself

Reply:
I'd be alright with dropping it.

Reply:
Removed.